dali/dfu/dfurun.cpp

```diff
@@ -1604,7 +1604,8 @@ class CDFUengine: public CInterface, implements IDFUengine
                         if (needrep)
                             feedback.repmode=cProgressReporter::REPbefore;
                         fsys.import(fdesc, dstFile, recovery, recoveryconn, filter, opttree, &feedback, &abortnotify, dfuwuid);
-                        if (!abortnotify.abortRequested()) {
+                        if (!abortnotify.abortRequested())
+                        {
                             if (needrep)
```

Comment:
I can't see it as it's been squashed, but I think before your latest change here, it was testing if no source parts..
This change to proceed an attach is good, but the needrep/replicating=true is not wanted if there's nothing to do, so it would be worth putting a check around that to avoid it.